definitions/divorce/json/AuthorisationCaseField/AuthorisationCaseField-deemed-and-dispensed-nonprod.json

```diff
@@ -279,6 +279,48 @@
     "UserRole": "caseworker-divorce-solicitor",
```

Comment:
For the 'ccd-definition-processor' was the change intentional? If not, you need to run 'yarn setup' (to pull the latest ccd-definition-processor) on your local machine and then push again.

Reply:
No change was not intentional, would do as recomended

Reply:
Local workspace seems to have no changes after running the command above.